gpbft/api.go

```diff
@@ -54,9 +54,11 @@ type MessageReceiver interface {
 
 // Interface which network participants must implement.
 type Receiver interface {
-	// Begins executing the protocol.
+	// Beings the execution of the instance number passed as an argument.
 	// The node will request the canonical chain to propose from the host.
-	Start() error
+	// This function cna be used to skip directly to an instance number,
+	// for instance, when a new valid finality certificate is received
```

Comment:
```suggestion
	// SkipToInstance jumps directly to a given instance when justified by a finality certificate.
```

...or something along those lines. At the very least fix typo `cna` and end with `.`.